### ISSUE TITLE
Standalone Tika server config

### ISSUE THREAD

**Issue by brianlmerritt** [mhughes2k/moodle-search_solrrag#2]
If I select a standalone Tika server I don't see where to enter the config details on the settings.  

Should that be an option?

**Comment by mhughes2k** [comment]
Yes this is a working progress, since the Solr tika isn't "recommended" for production.
